### ISSUE TITLE
Possible dependency issue with Microsoft.AspNet.Authentication.Google  1.0.0-rc2-16148

### ISSUE THREAD

**Issue by mikes-gh** [aspnet/Security#642]
If I add 

```
 app.UseGoogleAuthentication(options =>
            {
                options.ClientId = "***.apps.googleusercontent.com";
                options.ClientSecret = "***";
            });
```
to my startup Configure method I get

```
System.IO.FileNotFoundException was unhandled by user code
  FileName=Microsoft.Extensions.OptionsModel, Version=1.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60
  FusionLog==== Pre-bind state information ===
LOG: DisplayName = Microsoft.Extensions.OptionsModel, Version=1.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60
 (Fully-specified)
LOG: Appbase = file:///C:/Users/mikes/.dnx/runtimes/dnx-clr-win-x86.1.0.0-rc2-16357/bin/
LOG: Initial PrivatePath = NULL
Calling assembly : (Unknown).
===
LOG: This bind starts in default load context.
LOG: No application configuration file found.
LOG: Using host configuration file: 
LOG: Using machine configuration file from C:\Windows\Microsoft.NET\Framework\v4.0.30319\config\machine.config.
LOG: Post-policy reference: Microsoft.Extensions.OptionsModel, Version=1.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60
LOG: Fusion is hosted. Check host about this assembly.
LOG: Try host assembly store with assembly microsoft.extensions.optionsmodel, version=1.0.0.0, culture=neutral, publickeytoken=adb9793829ddae60, processorarchitecture=x86.
LOG: Try host assembly store with assembly microsoft.extensions.optionsmodel, version=1.0.0.0, culture=neutral, publickeytoken=adb9793829ddae60, processorarchitecture=msil.
LOG: Try host assembly store with assembly microsoft.extensions.optionsmodel, version=1.0.0.0, culture=neutral, publickeytoken=adb9793829ddae60.
WRN: Host assembly store does not contain this assembly.
LOG: Attempting download of new URL file:///C:/Users/mikes/.dnx/runtimes/dnx-clr-win-x86.1.0.0-rc2-16357/bin/Microsoft.Extensions.OptionsModel.DLL.
LOG: Attempting download of new URL file:///C:/Users/mikes/.dnx/runtimes/dnx-clr-win-x86.1.0.0-rc2-16357/bin/Microsoft.Extensions.OptionsModel/Microsoft.Extensions.OptionsModel.DLL.
LOG: Attempting download of new URL file:///C:/Users/mikes/.dnx/runtimes/dnx-clr-win-x86.1.0.0-rc2-16357/bin/Microsoft.Extensions.OptionsModel.EXE.
LOG: Attempting download of new URL file:///C:/Users/mikes/.dnx/runtimes/dnx-clr-win-x86.1.0.0-rc2-16357/bin/Microsoft.Extensions.OptionsModel/Microsoft.Extensions.OptionsModel.EXE.

  HResult=-2147024894
  Message=Could not load file or assembly 'Microsoft.Extensions.OptionsModel, Version=1.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60' or one of its dependencies. The system cannot find the file specified.
  Source=mscorlib
  StackTrace:
       at System.Signature.GetSignature(Void* pCorSig, Int32 cCorSig, RuntimeFieldHandleInternal fieldHandle, IRuntimeMethodInfo methodHandle, RuntimeType declaringType)
       at System.Signature..ctor(IRuntimeMethodInfo methodHandle, RuntimeType declaringType)
       at System.Reflection.RuntimeConstructorInfo.GetParametersNoCopy()
       at System.Reflection.RuntimeConstructorInfo.GetParameters()
       at Microsoft.Extensions.Internal.ActivatorUtilities.ConstructorMatcher..ctor(ConstructorInfo constructor)
       at Microsoft.Extensions.Internal.ActivatorUtilities.<>c.<CreateInstance>b__1_1(ConstructorInfo constructor)
       at System.Linq.Enumerable.WhereSelectArrayIterator`2.MoveNext()
       at Microsoft.Extensions.Internal.ActivatorUtilities.CreateInstance(IServiceProvider provider, Type instanceType, Object[] parameters)
       at Microsoft.AspNet.Builder.UseMiddlewareExtensions.<>c__DisplayClass2_0.<UseMiddleware>b__0(RequestDelegate next)
       at Microsoft.AspNet.Builder.Internal.ApplicationBuilder.Build()
       at Microsoft.AspNet.Builder.MapExtensions.Map(IApplicationBuilder app, PathString pathMatch, Action`1 configuration)
       at Reports2.Startup.Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerFactory loggerFactory) in C:\Users\mikes\Projects\Reports2\Main\Reports2\src\Reports2\Startup.cs:line 67
  InnerException: 
```
Facebook provider works ok.
Looks like a dependency on Extensions.OptionsModel but it looks like the renames have already been done.

Probably a result of 

https://github.com/aspnet/Announcements/issues/132

**Comment by Tratcher** [comment]
That's weird. Try running `dnu list --details` to see who is pulling in the old OptionsModel package.

**Comment by mikes-gh** [comment]
It was an old nugget package.  I used visual studio to install (click on the red squiggle).  That installed a fixed version that was lower than 1.0.0-* which gave 1.0.0-rc2-16216 which works fine
Sorry for noise
